be/src/Project/Services/DownloadCenter/DownloadCenter.Api/Services/DownloadCenter/DownloadCenterService.cs

```diff
@@ -223,28 +223,24 @@ public async Task<DownloadCenterCategoryServiceModel> GetDownloadCenterCategoryA
 
             var subcategories = _context.DownloadCenterCategories.Where(x => x.ParentCategoryId == downloadCenterCategory.Id).ToList();
 
-            var translations = _context.DownloadCenterCategoryTranslations.Where(x => x.CategoryId == downloadCenterCategory.Id || x.CategoryId == downloadCenterCategory.ParentCategoryId || subcategories.Select(y => y.Id).Contains(x.CategoryId)).ToList();
-
-            var files = _context.DownloadCenterCategoryFiles.Where(x => x.CategoryId == downloadCenterCategory.Id && x.IsActive);
-
             return new DownloadCenterCategoryServiceModel
             {
                 Id = downloadCenterCategory.Id,
-                CategoryName = translations.FirstOrDefault(t => t.CategoryId == model.Id && t.IsActive && t.Language == model.Language)?.Name ?? translations.FirstOrDefault(t => t.CategoryId == model.Id && t.IsActive)?.Name,
+                CategoryName = downloadCenterCategory.Translations.FirstOrDefault(t => t.CategoryId == model.Id && t.IsActive && t.Language == model.Language)?.Name ?? downloadCenterCategory.Translations.FirstOrDefault(t => t.CategoryId == model.Id && t.IsActive)?.Name,
```

Comment:
**suggestion:** if you don't add .Include and .AsSingleQuery statements it will result in many database calls